### ISSUE TITLE
unable to compile, error during compilation

### ISSUE THREAD

**Issue by Slushi-Github** [MAJigsaw77/hxvlc#55]
I have been trying to compile hxvlc, but it is impossible, during the compilation (which is from scratch, new compilation), this error occurs:
![Code_zFBbxvcOE8](https://github.com/MAJigsaw77/hxvlc/assets/81278332/f7bdbea3-fc63-4f86-9e17-fec98b4ff4e7)

yes, an apology that this is in Spanish the error hehe...

**Comment by MAJigsaw77** [comment]
If you somehow use hmm, be sure both libs are the same and be sure to also do a clean compile using -clean option

**Comment by Slushi-Github** [comment]
I installed it using hmm, re-installed it using ``haxelib git`` and nothing, I have tried ``-clean`` and nothing either, it doesn`t compile and I am forced to disable the video feature.

**Comment by MAJigsaw77** [comment]
Are you sure you don't use any another lib that does this? Maybe also try to delete the hole export

**Comment by Slushi-Github** [comment]
No, I am using libraries in local mode inside the repository I am using and there is only hxvlc and only hxvlc is imported in ``project.xml`` and of course where it is needed for video stuff.

**Comment by MAJigsaw77** [comment]
Try deleting the hole export

**Comment by Slushi-Github** [comment]
What does it mean? Try using ``-clean`` again?

**Comment by MAJigsaw77** [comment]
> What does it mean? Try using `-clean` again?

No, delete the folder

**Comment by Slushi-Github** [comment]
nope, nothing, the same error again.

**Comment by MAJigsaw77** [comment]
> nope, nothing, the same error again.

I suggest you to check again because the workflows doesn't seem to have an issue compiling

**Comment by Slushi-Github** [comment]
I finally found the error, it was HxCodec. 
I really don't understand how it interferes with a library that is not being imported into the project and it doesn't even exist locally in the project. 
Thanks for the help

**Comment by MAJigsaw77** [comment]
> I finally found the error, it was HxCodec. I really don't understand how it interferes with a library that is not being imported into the project and it doesn't even exist locally in the project. Thanks for the help

Np

**Comment by kornelbut** [comment]
> I finally found the error, it was HxCodec. I really don't understand how it interferes with a library that is not being imported into the project and it doesn't even exist locally in the project. Thanks for the help

I'm getting the same error at the moment. What did you do with hxCodec to fix it?

**Comment by kornelbut** [comment]
Alright thanks

**Comment by Zuk0109** [comment]
how exactly did you remove hxcodec? all i did was remove it from the project.xml file

**Comment by Slushi-Github** [comment]
> how exactly did you remove hxcodec? all i did was remove it from the project.xml file and i get a VideoHandler error now

``haxelib remove hxcodec``
